src/cljs/replumb/repl.cljs

```diff
@@ -592,6 +625,35 @@
       (call-back (common/wrap-success (s/join (map #(with-out-str (repl/print-doc %)) ms))))
       (call-back (common/wrap-success "nil")))))
 
+(defn last-form
+  [source]
+  (let [rdr (rt/string-push-back-reader source)
+        eof (js-obj)
+        read #(r/read {:eof eof} rdr)]
+    (loop [first-form (read)
+           second-form (read)]
+      (if (identical? eof second-form)
+        first-form
+        (recur second-form (read))))))
+
+(defn process-load-file
+  [{:keys [verbose read-file-fn! src-paths] :as opts} cb data filename]
+  (let [call-back (partial call-back! opts cb data)]
+    (if read-file-fn!
+      (load/read-files-and-callback! verbose
+                                     (load/file-paths src-paths filename)
+                                     read-file-fn!
+                                     (fn [result]
+                                       (if result
+                                         (let [eval-opts (load-eval-opts! opts filename)
+                                               src (:source result)]
+                                           (eval-str* eval-opts opts cb data src (last-form src)))
```

Comment:
isn't the `src` here actually corresponding with the `name` parameter (which is wrong) ? It looks to me you are evaluating only the `last-form` here but I might be very wrong (that is why the above test is a better judge)